.github/workflows/examples-linux-tv-casting-app.yaml

```diff
@@ -79,9 +73,3 @@ jobs:
                   ./scripts/run_in_build_env.sh \
                     "python3 ./scripts/tests/run_tv_casting_test.py --commissioner-generated-passcode=True"
               timeout-minutes: 2 # Comment this out to debug if GitHub Action times out.
-
-            - name: Uploading Size Reports
```

Comment:
This seems unrelated to this PR.  Same for a bunch of the other workflow changes.